.github/workflows/update-docs.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         working-directory: ./tauri/core/tauri
         run: cargo doc --no-deps
       - name: run rustdocusaurus
-        uses: tauri-apps/rustdocusaurus/github-action@v1
+        uses: tauri-apps/rustdocusaurus/github-action@v1.0.3
```

Comment:
you should probably do the same as covector and update the v1 tag every time a new patch is released, but let's do it this way for now.